app/src/include/firebase/cpp_warning.h

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Could we rename this to cpp_version_warning.h? Just so it's totally clear what it does. (Or even cpp14_warning.h would also be good.)